docs/csharp/language-reference/builtin-types/struct.md

```diff
@@ -67,7 +67,7 @@ Typically, you apply the `readonly` modifier to the following kinds of instance
 
   :::code language="csharp" source="snippets/shared/StructType.cs" id="ReadonlyWithInit":::
 
-You can't apply the `readonly` modifier to static members of a structure type.
+You can apply the `readonly` modifier to static members of a structure type excluding static members able to modify the structure instance.
```

Comment:
Thanks for adding your comments @chessboards. That helps. I'd suggest updating this PR as follows:

```suggestion
You can apply the `readonly` modifier to static fields of a structure type, but not any other static members, such as properties or methods.
```